zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -752,10 +756,7 @@ function PageWrapper(props) {
                   </InputLabel>
                   <Autocomplete
                     options={options}
-                    defaultValue={
-                      props.location.search &&
-                      getQueryParams(window.location.href).get('q')
-                    }
+                    defaultValue={getQueryParams(window.location.href).get('q')}
```

Comment:
Why remove the props here?

Reply:
I think I made this change unintentionally while trying to figure out the working of the code. 

I will reset the change in a new commit and make sure to go through the changes I make next time.